domain_access/domain_access.module

```diff
@@ -25,21 +26,24 @@ define('DOMAIN_ACCESS_USER_FIELD', 'field_domain_user');
 /**
  * Implements hook_node_grants().
  */
-function domain_access_node_grants($op, AccountInterface $account) {
+function domain_access_node_grants(AccountProxy $account, $op) {
```

Comment:
No. AccountProxy is an implementation of AcoutnInterface. It is proper to type hint with the Interface, because implementations may change.

This has been accounted for over in PR #37